osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -616,6 +607,23 @@ - (void)openFunction {
 }
 
 
+- (IBAction)quitApplcation:(id)sender {
```

Comment:
Is this the correct spelling for the Action?  Just concerned that perhaps it's `quitApplication` but since it runs, maybe just a typo.